tutorials/get-started-canonical-kubernetes/get-started-canonical-kubernetes.md

```diff
@@ -1,187 +1,179 @@
- 
- 
+
+- Get your Kubernetes cluster up and running
+- Open the K8s dashboard
+- Control your cluster from the kubectl CLI client
+- Deploy your first container workload
+- Add extra features to your Kubernetes cluster
+
+
 ### You will need...
- 
-* An [Ubuntu One account](https://login.ubuntu.com/).
-* An SSH key in your JAAS account. If you haven’t done this already, [learn how to add your SSH key](https://jujucharms.com/docs/2.1/users-auth#credentials-and-ssh-keys).
-* Basic knowledge of command line.
- 
- 
- 
+
+* An [Ubuntu One account](https://login.ubuntu.com/)
+* An SSH key. [More on using SSH keys in Juju](https://jujucharms.com/docs/2.1/users-auth#credentials-and-ssh-keys).
+* Basic knowledge of command line
+
+
+
 ## Deploying your model with JAAS
 Duration: 1:00
- 
-With JAAS, you don’t need to deal with the setup and configuration of Juju. All you will need is your preferred cloud credentials.
- 
-You will need to deploy [The Canonical Distribution of Kubernetes](https://jujucharms.com/new?dd=bundle/canonical-kubernetes) before you get started. 
- 
+
+With JAAS, you don’t need to deal with the setup and configuration of Juju. All you will need is credentials for AWS, GCE, or Azure.
+
+Kick off deployment of your Kubernetes cluster by clicking the follow link: [The Canonical Distribution of Kubernetes](https://jujucharms.com/new?dd=bundle/canonical-kubernetes).
+
 Deployment will take several minutes as Juju creates new instances in the cloud and sets up the Kubernetes cluster components. Pending units are outlined in *orange*. Up and running ones are outlined in *black*.
 ![Status GUI](./images/status-gui.png)
- 
- 
- 
+
+
+
 ## Getting started
 Duration: 5:00
- 
+
 ### Install the Juju client
- 
+
 **You will need to have Juju installed locally to operate your cluster. Skip this step if you already have it.**
- 
+
 Juju is available as a client on many platforms and distributions. Regardless of your operating system, getting all the required tools is very easy. Visit the [install docs](https://jujucharms.com/docs/stable/reference-install#getting-the-latest-juju) to get the latest version of Juju on **macOS**, **Windows** or **CentOS**.
- 
+
 If you are running **Ubuntu**, you can install Juju through the following steps:
- 
-1. It’s helpful to Install Snappy if you don’t have it already. 
+
+1. It’s helpful to Install Snappy if you don’t have it already.
 `$ sudo apt install snapd`
- 
-2. Install Juju to get the command line client. 
+2. Install Juju to get the command line client.
 `$ sudo snap install juju --classic`
- 
 3. Verify you can run Juju. You will see a summary and a list of common commands.
 `$ juju`
- 
-You’re done! To learn more on how to get started with Juju, visit the [documentation](https://jujucharms.com/docs/stable/getting-started). 
- 
-For general help in the CLI: 
+
+You’re done! To learn more on how to get started with Juju, visit the [documentation](https://jujucharms.com/docs/stable/getting-started).
```

Comment:
remove :"You're done" - they've only just started...

Learn more about [getting started with Juju](....)